Build failure: bruno

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

`nix build nixpkgs/946738808f0ddc9e57ee0df76aa5986715c24676#bruno`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/317975877

### Relevant log output

```console
Running phase: buildPhase
npm warn Unknown env config "nodedir". This will stop working in the next major version of npm.
npm warn Unknown env config "platform". This will stop working in the next major version of npm.
npm warn Unknown env config "arch". This will stop working in the next major version of npm.
npm error code 1
npm error path /build/source/node_modules/canvas
npm error command failed
npm error command sh -c node-pre-gyp install --fallback-to-build --update-binary
npm error make: Entering directory '/build/source/node_modules/canvas/build'
npm error   SOLINK_MODULE(target) Release/obj.target/canvas-postbuild.node
npm error   COPY Release/canvas-postbuild.node
npm error   CXX(target) Release/obj.target/canvas/src/backend/Backend.o
npm error make: Leaving directory '/build/source/node_modules/canvas/build'
npm error Failed to execute '/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/bin/node /nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --update-binary --module=/build/source/node_modules/canvas/build/Release/canvas.node --module_name=canvas --module_path=/build/source/node_modules/canvas/build/Release --napi_version=10 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v137' (1)
npm error node-pre-gyp info it worked if it ends with ok
npm error node-pre-gyp info using node-pre-gyp@1.0.11
npm error node-pre-gyp info using node@24.12.0 | linux | x64
npm error node-pre-gyp http GET https://github.com/Automattic/node-canvas/releases/download/v2.11.2/canvas-v2.11.2-node-v137-linux-glibc-x64.tar.gz
npm error node-pre-gyp ERR! install request to https://github.com/Automattic/node-canvas/releases/download/v2.11.2/canvas-v2.11.2-node-v137-linux-glibc-x64.tar.gz failed, reason: getaddrinfo EAI_AGAIN github.com 
npm error node-pre-gyp WARN Pre-built binaries not installable for canvas@2.11.2 and node@24.12.0 (node-v137 ABI, glibc) (falling back to source compile with node-gyp) 
npm error node-pre-gyp WARN Hit error request to https://github.com/Automattic/node-canvas/releases/download/v2.11.2/canvas-v2.11.2-node-v137-linux-glibc-x64.tar.gz failed, reason: getaddrinfo EAI_AGAIN github.com 
npm error gyp info it worked if it ends with ok
npm error gyp info using node-gyp@11.4.2
npm error gyp info using node@24.12.0 | linux | x64
npm error gyp info ok 
npm error gyp info it worked if it ends with ok
npm error gyp info using node-gyp@11.4.2
npm error gyp info using node@24.12.0 | linux | x64
npm error gyp info find Python using Python version 3.13.11 found at "/nix/store/qzc04a3npl70cyyy6flnnrb2ig3kayxm-python3-3.13.11/bin/python3"
npm error gyp WARN read config.gypi ENOENT: no such file or directory, open '/nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/include/node/config.gypi'
npm error gyp info spawn /nix/store/qzc04a3npl70cyyy6flnnrb2ig3kayxm-python3-3.13.11/bin/python3
npm error gyp info spawn args [
npm error gyp info spawn args '/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm error gyp info spawn args 'binding.gyp',
npm error gyp info spawn args '-f',
npm error gyp info spawn args 'make',
npm error gyp info spawn args '-I',
npm error gyp info spawn args '/build/source/node_modules/canvas/build/config.gypi',
npm error gyp info spawn args '-I',
npm error gyp info spawn args '/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm error gyp info spawn args '-I',
npm error gyp info spawn args '/nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/common.gypi',
npm error gyp info spawn args '-Dlibrary=shared_library',
npm error gyp info spawn args '-Dvisibility=default',
npm error gyp info spawn args '-Dnode_root_dir=/nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source',
npm error gyp info spawn args '-Dnode_gyp_dir=/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/lib/node_modules/npm/node_modules/node-gyp',
npm error gyp info spawn args '-Dnode_lib_file=/nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/$(Configuration)/node.lib',
npm error gyp info spawn args '-Dmodule_root_dir=/build/source/node_modules/canvas',
npm error gyp info spawn args '-Dnode_engine=v8',
npm error gyp info spawn args '--depth=.',
npm error gyp info spawn args '--no-parallel',
npm error gyp info spawn args '--generator-output',
npm error gyp info spawn args 'build',
npm error gyp info spawn args '-Goutput_dir=.'
npm error gyp info spawn args ]
npm error gyp info ok 
npm error gyp info it worked if it ends with ok
npm error gyp info using node-gyp@11.4.2
npm error gyp info using node@24.12.0 | linux | x64
npm error gyp info spawn make
npm error gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm error In file included from ../src/backend/Backend.h:6,
npm error                  from ../src/backend/Backend.cc:1:
npm error ../../nan/nan.h: In constructor 'Nan::Utf8String::Utf8String(v8::Local<v8::Value>)':
npm error ../../nan/nan.h:1173:36: warning: 'int v8::String::WriteUtf8(v8::Isolate*, char*, int, int*, int) const' is deprecated: Use WriteUtf8V2 instead. [-Wdeprecated-declarations]
npm error  1173 |         length_ = string->WriteUtf8(v8::Isolate::GetCurrent(), str_,
npm error       |                   ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
npm error  1174 |                                     static_cast<int>(len), 0, flags);
npm error       |                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
npm error In file included from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-object.h:12,
npm error                  from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-array-buffer.h:14,
npm error                  from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8.h:24,
npm error                  from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/src/node.h:74,
npm error                  from ../../nan/nan.h:62:
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-primitive.h:212:7: note: declared here
npm error   212 |   int WriteUtf8(Isolate* isolate, char* buffer, int length = -1,
npm error       |       ^~~~~~~~~
npm error ../../nan/nan.h: In function 'v8::Local<v8::Value> Nan::Encode(const void*, size_t, Encoding)':
npm error ../../nan/nan.h:2418:24: warning: 'v8::Local<v8::Value> node::Encode(v8::Isolate*, const uint16_t*, size_t)' is deprecated: Use TryEncode(...) instead [-Wdeprecated-declarations]
npm error  2418 |     return node::Encode(
npm error       |            ~~~~~~~~~~~~^
npm error  2419 |         isolate
npm error       |         ~~~~~~~         
npm error  2420 |       , reinterpret_cast<const uint16_t *>(buf)
npm error       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
npm error  2421 |       , len / 2);
npm error       |       ~~~~~~~~~~        
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/src/node.h:1188:50: note: declared here
npm error  1188 |                 NODE_EXTERN v8::Local<v8::Value> Encode(v8::Isolate* isolate,
npm error       |                                                  ^~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/src/node.h:116:42: note: in definition of macro 'NODE_DEPRECATED'
npm error   116 |     __attribute__((deprecated(message))) declarator
npm error       |                                          ^~~~~~~~~~
npm error ../../nan/nan.h:2423:24: warning: 'v8::Local<v8::Value> node::Encode(v8::Isolate*, const char*, size_t, encoding)' is deprecated: Use TryEncode(...) instead [-Wdeprecated-declarations]
npm error  2423 |     return node::Encode(
npm error       |            ~~~~~~~~~~~~^
npm error  2424 |         isolate
npm error       |         ~~~~~~~         
npm error  2425 |       , reinterpret_cast<const char *>(buf)
npm error       |       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
npm error  2426 |       , len
npm error       |       ~~~~~             
npm error  2427 |       , node_enc);
npm error       |       ~~~~~~~~~~~       
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/src/node.h:1180:38: note: declared here
npm error  1180 |     NODE_EXTERN v8::Local<v8::Value> Encode(v8::Isolate* isolate,
npm error       |                                      ^~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/src/node.h:116:42: note: in definition of macro 'NODE_DEPRECATED'
npm error   116 |     __attribute__((deprecated(message))) declarator
npm error       |                                          ^~~~~~~~~~
npm error In file included from ../../nan/nan.h:3045:
npm error ../../nan/nan_scriptorigin.h: In constructor 'Nan::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>)':
npm error ../../nan/nan_scriptorigin.h:36:55: error: no matching function for call to 'v8::ScriptOrigin::ScriptOrigin(v8::Isolate*, v8::Local<v8::Value>&)'
npm error    36 |       v8::ScriptOrigin(v8::Isolate::GetCurrent(), name) {}
npm error       |                                                       ^
npm error ../../nan/nan_scriptorigin.h:36:55: note: there are 3 candidates
npm error In file included from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-script.h:20,
npm error                  from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-debug.h:10,
npm error                  from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-isolate.h:19,
npm error                  from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-snapshot.h:9,
npm error                  from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-context.h:15,
npm error                  from /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8.h:26:
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:65:13: note: candidate 1: 'v8::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, int, int, bool, int, v8::Local<v8::Value>, bool, bool, bool, v8::Local<v8::Data>)'
npm error    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
npm error       |             ^~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:65:39: note: no known conversion for argument 1 from 'v8::Isolate*' to 'v8::Local<v8::Value>'
npm error    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
npm error       |                          ~~~~~~~~~~~~~^~~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate 2: 'constexpr v8::ScriptOrigin::ScriptOrigin(const v8::ScriptOrigin&)'
npm error    63 | class V8_EXPORT ScriptOrigin {
npm error       |                 ^~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate expects 1 argument, 2 provided
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate 3: 'constexpr v8::ScriptOrigin::ScriptOrigin(v8::ScriptOrigin&&)'
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate expects 1 argument, 2 provided
npm error ../../nan/nan_scriptorigin.h: In constructor 'Nan::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, v8::Local<v8::Integer>)':
npm error ../../nan/nan_scriptorigin.h:42:52: error: no matching function for call to 'v8::ScriptOrigin::ScriptOrigin(v8::Isolate*, v8::Local<v8::Value>&, int)'
npm error    42 |                    , To<int32_t>(line).FromMaybe(0)) {}
npm error       |                                                    ^
npm error ../../nan/nan_scriptorigin.h:42:52: note: there are 3 candidates
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:65:13: note: candidate 1: 'v8::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, int, int, bool, int, v8::Local<v8::Value>, bool, bool, bool, v8::Local<v8::Data>)'
npm error    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
npm error       |             ^~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:65:39: note: no known conversion for argument 1 from 'v8::Isolate*' to 'v8::Local<v8::Value>'
npm error    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
npm error       |                          ~~~~~~~~~~~~~^~~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate 2: 'constexpr v8::ScriptOrigin::ScriptOrigin(const v8::ScriptOrigin&)'
npm error    63 | class V8_EXPORT ScriptOrigin {
npm error       |                 ^~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate expects 1 argument, 3 provided
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate 3: 'constexpr v8::ScriptOrigin::ScriptOrigin(v8::ScriptOrigin&&)'
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate expects 1 argument, 3 provided
npm error ../../nan/nan_scriptorigin.h: In constructor 'Nan::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, v8::Local<v8::Integer>, v8::Local<v8::Integer>)':
npm error ../../nan/nan_scriptorigin.h:50:54: error: no matching function for call to 'v8::ScriptOrigin::ScriptOrigin(v8::Isolate*, v8::Local<v8::Value>&, int, int)'
npm error    50 |                    , To<int32_t>(column).FromMaybe(0)) {}
npm error       |                                                      ^
npm error ../../nan/nan_scriptorigin.h:50:54: note: there are 3 candidates
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:65:13: note: candidate 1: 'v8::ScriptOrigin::ScriptOrigin(v8::Local<v8::Value>, int, int, bool, int, v8::Local<v8::Value>, bool, bool, bool, v8::Local<v8::Data>)'
npm error    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
npm error       |             ^~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:65:39: note: no known conversion for argument 1 from 'v8::Isolate*' to 'v8::Local<v8::Value>'
npm error    65 |   V8_INLINE ScriptOrigin(Local<Value> resource_name,
npm error       |                          ~~~~~~~~~~~~~^~~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate 2: 'constexpr v8::ScriptOrigin::ScriptOrigin(const v8::ScriptOrigin&)'
npm error    63 | class V8_EXPORT ScriptOrigin {
npm error       |                 ^~~~~~~~~~~~
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate expects 1 argument, 4 provided
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate 3: 'constexpr v8::ScriptOrigin::ScriptOrigin(v8::ScriptOrigin&&)'
npm error /nix/store/pq5996xinzcns0k5la4xhmlwzwgz2w6s-nodejs-24.12.0-source/deps/v8/include/v8-message.h:63:17: note: candidate expects 1 argument, 4 provided
npm error make: *** [canvas.target.mk:139: Release/obj.target/canvas/src/backend/Backend.o] Error 1
npm error gyp ERR! build error 
npm error gyp ERR! stack Error: `make` failed with exit code: 2
npm error gyp ERR! stack at ChildProcess.<anonymous> (/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:219:23)
npm error gyp ERR! System Linux 6.12.63
npm error gyp ERR! command "/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/bin/node" "/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--update-binary" "--module=/build/source/node_modules/canvas/build/Release/canvas.node" "--module_name=canvas" "--module_path=/build/source/node_modules/canvas/build/Release" "--napi_version=10" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v137"
npm error gyp ERR! cwd /build/source/node_modules/canvas
npm error gyp ERR! node -v v24.12.0
npm error gyp ERR! node-gyp -v v11.4.2
npm error gyp ERR! not ok 
npm error node-pre-gyp ERR! build error 
npm error node-pre-gyp ERR! stack Error: Failed to execute '/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/bin/node /nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --update-binary --module=/build/source/node_modules/canvas/build/Release/canvas.node --module_name=canvas --module_path=/build/source/node_modules/canvas/build/Release --napi_version=10 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v137' (1)
npm error node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/build/source/node_modules/@mapbox/node-pre-gyp/lib/util/compile.js:89:23)
npm error node-pre-gyp ERR! stack     at ChildProcess.emit (node:events:508:28)
npm error node-pre-gyp ERR! stack     at maybeClose (node:internal/child_process:1101:16)
npm error node-pre-gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:305:5)
npm error node-pre-gyp ERR! System Linux 6.12.63
npm error node-pre-gyp ERR! command "/nix/store/cikdc61gfwvdma6y0p9b5d5d448aqcv6-nodejs-24.12.0/bin/node" "/build/source/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build" "--update-binary"
npm error node-pre-gyp ERR! cwd /build/source/node_modules/canvas
npm error node-pre-gyp ERR! node -v v24.12.0
npm error node-pre-gyp ERR! node-pre-gyp -v v1.0.11
npm error node-pre-gyp ERR! not ok
npm error Log files were not written due to an error writing to the directory: /nix/store/c8m6a9qdx3j5cxmm2byb6q4cmafr72lv-bruno-2.15.1-npm-deps/_logs
npm error You can rerun the command with `--loglevel=verbose` to see the logs in your terminal
```

### Additional context

Possibly https://github.com/NixOS/nixpkgs/issues/474535

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.18.1-zen1, NixOS, 26.05 (Yarara), 26.05.20251228.c0b0e0f`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.93.3
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux, x86_64-v4-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/gep/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/gep/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/gep/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/gep/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/sa7kxynphvibmp8z8dar31j3dyn3nvfj-lix-2.93.3/share`
 - nixpkgs: `/nix/store/1ny7brxnqbx6xilj7mjdlinzpb5a1s3i-source`

### Notify maintainers

@gepbird @kashw2 @lucasew @mattpolzin @redyf @water-sucks @starsep 

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/br/bruno/package.nix
+++ b/pkgs/by-name/br/bruno/package.nix
@@ -5,6 +5,7 @@
   fetchFromGitHub,
   buildNpmPackage,
   nix-update-script,
+  nodejs_22,
   electron,
   makeWrapper,
   copyDesktopItems,
@@ -32,6 +33,8 @@ buildNpmPackage rec {
       ${lib.getExe npm-lockfile-fix} $out/package-lock.json
     '';
   };
+
+  nodejs = nodejs_22;
 
   npmDepsHash = "sha256-CXXXyDaaoAoZhUo1YNP3PUEGzlmIaDnA+JhrCqBY1H4=";
   npmFlags = [ "--legacy-peer-deps" ];

